src/components/RenderInfringement/RenderInfringment.jsx

```diff
@@ -4,11 +4,11 @@
 // import _ from 'lodash'
 // import ShowSaveWarning from "../common/ShowSaveWarning"
 
-// class RenderInfringment extends Form {
+// class Renderinfringement extends Form {
```

Comment:
There are multiple names not using camelCase in this file. 
This file name is also misspelled.
Please check it, thank you!

Reply:
Thanks for the catch. Fixed!